### DISCUSSION TITLE
I'm trying to use `Shift+Return`

### DISCUSSION THREAD

**Discussion by moshpirit** [davatorium/rofi#2064] [Q&A]
I don't know why, but I can't use `shift+return` as an alt+keymap. 
I'm trying `echo -e "Option 1\nOption 2" | rofi -dmenu -p "Test" -kb-accept-alt "Shift+Return"; echo "Output code: $?"`, but this is what I get when I run it and press `shift+return`:
```
Option 1
Output code: 0
```

**Comment by DaveDavenport** [comment]
What behaviour do you expect to happen?

**Reply by moshpirit** [reply]
I want to be able to modify the entries of the clipboard. I'm testing this code that I got from chatgpt (I know it's not ideal, but I'm not a developer); pressing `<SR>` should return option 10 according to it (again, I am no dev, I'm just trying to learn)

```bash
#!/bin/bash

# Ejecutar rofi con greenclip y capturar la selección
SELECTION=$(rofi -modi "clipboard:greenclip print" -show clipboard -config ~/.config/rofi/launchers/type-1/clipboard.rasi)
EXIT_CODE=$? # Captura el código de salida de rofi

# Manejar acciones dependiendo del código de salida
case $EXIT_CODE in
    0) # Return (Enter)
        echo "Seleccionaste: $SELECTION"
        ;;
    10) # Shift + Return (kb-accept-alt)
        echo "$SELECTION" | xclip -selection clipboard
        NEW_TEXT=$(echo "$SELECTION" | rofi -dmenu -p "Editar texto del clipboard")
        if [[ -n "$NEW_TEXT" ]]; then
            echo "$NEW_TEXT" | xclip -selection clipboard
            notify-send "Clipboard actualizado" "$NEW_TEXT"
        fi
        ;;
    *)
        echo "Acción cancelada."
        ;;
esac
```

**Reply by DaveDavenport** [reply]
Yes, chatgpt is wrong/incomplete. Why not check the manual?

**Reply by moshpirit** [reply]
I have checked the manual, but I don't know how to use that information (in this case [scripts](https://davatorium.github.io/rofi/current/rofi-script.5/) and [keys](https://davatorium.github.io/rofi/current/rofi-keys.5/)) for what I want (editing the clipboard), tbh.

**Reply by DaveDavenport** [reply]
You are using rofi in dmenu mode, so I would read the [dmenu](https://davatorium.github.io/rofi/current/rofi-dmenu.5/#return-value) manpage.

**Reply by DaveDavenport** [reply]
If you want to have different return code on certain bindings, you need to use the 'custom-keys' (kb-custom-*)
